plugins/inputs/nfs/nfs.go

```diff
@@ -0,0 +1,344 @@
+package nfs
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+type NFS struct {
+	Iostat   bool
+	Fullstat bool
+}
+
+var sampleConfig = `
```

Comment:
Everything that is shown in mountstats, with fullstats set true, the plugin parses and send all data as is.

The iostat parses just a few common used data and some calculation is made as well.

The differences between setting each to true is at the readme, on the Measurements & Fields section and what each field really means are on the Reference section.

For example, in the fullstat It's shown how many cached bytes have been written or read. iostat doesn't do that.

It really depends on what you are trying to do. And both can be set to true, since each option gives different outputs. Although getting the fullstat and making those calculations that iostat does, works as well but a little harder.